README.md

```
@@ -1040,6 +1040,18 @@ relative links inside lesson docs.
 ░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒░░░▒▒▒
 ```
 
+## Translations
+
+Community-maintained adaptations of this curriculum:
+
+- 🇹🇷 **Turkish** — [komunite/ai-engineering](https://github.com/komunite/ai-engineering) · maintained by [Komünite](https://komunite.com.tr) · site: <https://ai-muhendisligi.komunite.com.tr>
+
+Maintain a translation in another language? Open a PR adding it here, following the [FORKING.md](FORKING.md) guidance under "For Other Languages".
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)